aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -110,41 +110,53 @@ To configure an app for the Azure SignalR Service, the app must support *sticky
   });
   ```
 
-* Configuration (use **one** of the following approaches):
+* Configure the option in `appsettings.json`:
 
-  * In `appsettings.json`:
-
-    ```json
-    "Azure:SignalR:ServerStickyMode": "Required"
-    ```
-
-  * The app service's **Configuration** > **Application settings** in the Azure portal (**Name**: `Azure__SignalR__ServerStickyMode`, **Value**: `Required`). This approach is adopted for the app automatically if you [provision the Azure SignalR Service](#provision-the-azure-signalr-service).
+  ```json
+  "Azure:SignalR:ServerStickyMode": "Required"
+  ```
 
 > [!NOTE]
-> The following error is thrown by an app that hasn't enabled sticky sessions for Azure SignalR Service:
+> The following error is thrown by an app that hasn't enabled session affinity:
 >
-> > blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.
-
-### Provision the Azure SignalR Service
+> > :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
 
 To provision the Azure SignalR Service for an app in Visual Studio:
 
 1. Create an Azure Apps publish profile in Visual Studio for the app.
 1. Add the **Azure SignalR Service** dependency to the profile. If the Azure subscription doesn't have a pre-existing Azure SignalR Service instance to assign to the app, select **Create a new Azure SignalR Service instance** to provision a new service instance.
 1. Publish the app to Azure.
 
-Provisioning the Azure SignalR Service in Visual Studio automatically [enables *sticky sessions*](#configuration) and adds the SignalR connection string to the app service's configuration.
+Provisioning the Azure SignalR Service in Visual Studio automatically adds the SignalR connection string to the app service's configuration.
+
+## Azure App Service without the Azure SignalR Service
```

Comment:
I think the section names for the Azure App Service section and the Azure Container Apps section should consistent, and the "without Azure SignalR Service" bit doesn't really seem necessary, so I suggest removing it.

```suggestion
## Azure App Service
```